### PR TITLE
Adding endpoint info to network inspect

### PR DESCRIPTION
**Pull request by hmahmood**

### CODE REVIEW

**Comment by hickeng** [comment]
Will need to get the tether tests compiling and passing before we put any else into it.

**Comment by caglar10ur** [comment]
LGTM

File: lib/apiservers/portlayer/restapi/handlers/scopes_handlers.go [hunk 1]
```diff
@@ -259,17 +293,48 @@ func (handler *ScopesHandlersImpl) ScopesUnbindContainer(params scopes.UnbindCon
...
+		sc.Endpoints[i] = &models.EndpointConfig{
+			Gateway:   e.Gateway().String(),
+			ID:        e.ID(),
+			Name:      e.Container().Name(),
+			Network:   e.Scope().Name(),
+			Container: e.Container().ID().String(),
+		}
+
+		ip := e.IP()
+		if ip != nil {
```
Comment (hickeng) [inline lib/apiservers/portlayer/restapi/handlers/scopes_handlers.go]: If you move this block above line 324 and assign Address in the creation it's easier to read. addr can be "" by default for the case ip == nil.

File: lib/apiservers/portlayer/restapi/handlers/scopes_handlers.go [hunk 2]
```diff
@@ -259,17 +293,48 @@ func (handler *ScopesHandlersImpl) ScopesUnbindContainer(params scopes.UnbindCon
...
+		// use subnet as pool
+		sc.IPAM = []string{subnet}
+	}
+
+	eps := scope.Endpoints()
+	sc.Endpoints = make([]*models.EndpointConfig, len(eps))
+	for i, e := range eps {
+		sc.Endpoints[i] = &models.EndpointConfig{
+			Gateway:   e.Gateway().String(),
+			ID:        e.ID(),
```
Comment (hickeng) [inline lib/apiservers/portlayer/restapi/handlers/scopes_handlers.go]: I should know already - what's is the value of this ID given we're not tracking endpoints in the same way docker is?

**Comment by hickeng** [comment]
Minor comments, LGTM.

File: lib/apiservers/portlayer/restapi/handlers/scopes_handlers.go (hunk 2 above)
Comment (hmahmood) [inline lib/apiservers/portlayer/restapi/handlers/scopes_handlers.go]: This is the PL endpoint struct, so we do track those like docker. The id is just a randomly generated string.

**Comment by hmahmood** [comment]
♻️

**Pull request merged**